packages/core/src/httpClient/retryableHttpClient.ts

```diff
@@ -38,8 +39,8 @@ import { MAX_RETRY_ATTEMPTS, type RetryValidator } from './retryValidator';
  */
 export class RetryableHttpClient extends BasicHttpClient {
   private static calculateRetryDelayInMs(retryCount: number) {
-    const INITIAL_RETRY_DELAY_IN_MS = 250;
-    return INITIAL_RETRY_DELAY_IN_MS + ((2 ** retryCount - 1) / 2) * 1000;
+    const backoff = new ExponentialJitterBackoff();
```

Comment:
We initialize a new class for each call to `calculateRetryDelayInMs`?
Should we reuse the class instance (which seems safe to do) or convert the method to a static method?